### PR TITLE
New Update

### PR DESCRIPTION
**Pull request by AzerothA**

### CODE REVIEW

File: src/ui/page/page.module.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.page {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-end;
+    width: 100%;
+    height: 100%;
```
Comment (nexnema) [inline src/ui/page/page.module.css:6]: don't use height 100%, never ever.

File: src/ui/theme/useTheme.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React, { useEffect, useState } from "react";
+
+/**
+ * Sets class to html element
+ * @param styles Styles for colors (Css variables)
+ * @returns
+ */
+function useTheme(styles: string) {
```
Comment (nexnema) [inline src/ui/theme/useTheme.tsx:8]: Explain me this function.

File: src/index.tsx [hunk 3]
```diff
@@ -4,7 +4,7 @@ import * as serviceWorkerRegistration from "./serviceWorkerRegistration";
 import reportWebVitals from "./reportWebVitals";
 import Router from "./router";
 import "./assets/css/style.css";
-import "./assets/css/color.css";
+// import "./assets/css/color.css";
```
Comment (nexnema) [inline src/index.tsx:7]: Why you comment this line?

**Review by nexnema — CHANGES_REQUESTED** [review]
Check comments, fix or explain them. thank you.

File: src/ui/button/button.tsx [hunk 4]
```diff
@@ -12,10 +12,14 @@ function Button(props: ButtonProps) {
             ? styles.buttonOutlined
             : styles.buttonText;
 
-    const classes = classnames(styles.button, variantStyle, props.className!);
+    const classNames = joinClassNames(
+        styles.button,
+        variantStyle,
```
[the remark this answers is not in the document]
Comment (AzerothA) [reply src/ui/button/button.tsx:17]: Join the styles to a one string

File: src/ui/theme/useTheme.tsx (hunk 2 above)
Comment (AzerothA) [reply src/ui/theme/useTheme.tsx:8]: Adds a variables class to the HTML tag.
And uses the effect to do that.

Maybe we can use a better way.

File: src/index.tsx (hunk 3 above)
Comment (AzerothA) [reply src/index.tsx:7]: because it's not an efficient way to use Theme.

Comment (nexnema) [reply src/index.tsx:7]: then, remove it.

**Pull request merged**